cli/tests/integrations/test_job.py

```diff
@@ -0,0 +1,175 @@
+import contextlib
+import json
+import os
+import re
+import sys
+import threading
+
+from dcos import constants
+
+import pytest
+from six.moves.BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+from .common import (app, job, show_job, assert_command, assert_lines, config_set,
+                     config_unset, exec_command, popen_tty, show_job_schedule,
+                      update_config, watch_all_deployments)
+
+def test_help():
+    with open('tests/data/help/job.txt') as content:
+        assert_command(['dcos', 'job', '--help'],
+                       stdout=content.read().encode('utf-8'))
+
+
+def test_version():
+    assert_command(['dcos', 'job', '--version'],
+                   stdout=b'dcos-job version SNAPSHOT\n')
+
+
+def test_info():
+    assert_command(['dcos', 'marathon', '--info'],
+                   stdout=b'Deploy and manage applications to DC/OS\n')
+
+
+@pytest.fixture
+def env():
+    r = os.environ.copy()
+    r.update({
+        constants.PATH_ENV: os.environ[constants.PATH_ENV],
+        constants.DCOS_CONFIG_ENV: os.path.join("tests", "data", "dcos.toml"),
+    })
+
+    return r
+
+
+def test_missing_config(env):
+    with update_config("core.dcos_url", None, env):
+        assert_command(
+            ['dcos', 'job', 'list'],
+            returncode=1,
+            stderr=(b'Missing required config parameter: "core.dcos_url".  '
+                    b'Please run `dcos config set core.dcos_url <value>`.\n'),
+            env=env)
```

Comment:
Yes, but this is already tested in the `config` test. Is there something specific to job here?